### PR TITLE
Fix Travis License Test

### LINKED ISSUE #152: Travis License Test not Working
Travis is passing in this PR https://github.com/omichel/webots/pull/149 where I did forget the license to the python controller.

File: https://github.com/omichel/webots/pull/149/files#diff-50972ec0b39faa5ffd49a8971161d27a
Travis test: https://travis-ci.org/omichel/webots/builds/488537485?utm_source=github_status&utm_medium=notification

### PR DESCRIPTION
**Pull request by DavidMansolino**
Fix #152

### CODE REVIEW

**Comment by DavidMansolino** [comment]
On travis the repository is located in `/home/travis/build/omichel/webots` and since we are skipping all the `build` directories all no file is tested at all. I will fix this.

**Comment by DavidMansolino** [comment]
The exact same issue was also present in the clang-format test.

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

File: src/webots/gui/WbWrenWindow.cpp [hunk 1]
```diff
@@ -290,7 +288,8 @@ void WbWrenWindow::flipAndScaleDownImageBuffer(const unsigned char *source, unsi
   const int w = sourceWidth / scaleDownFactor;
   const int yFactor = scaleDownFactor * sourceWidth;
 
-  // - The `unsigned char *` to `int *` cast is possible assuming that a pixel is coded as four bytes (RGBA) aligned on an `int *` boundary.
+  // - The `unsigned char *` to `int *` cast is possible assuming that a pixel is coded as four bytes (RGBA) aligned on an `int
+  // *` boundary.
```
Comment (omichel) [inline src/webots/gui/WbWrenWindow.cpp]: The line cut is not nice here...

**Comment by DavidMansolino** [comment]
:+1:    (thank you for the improvement)

**Comment by fabienrohrer** [comment]
Well spotted.
This leads to new errors in the current PRs.

**Comment by DavidMansolino** [comment]
> This leads to new errors in the current PRs.

Yes, that was to be expected.

**Pull request merged**